### ISSUE TITLE
JournalToSQLTable combined with EnsureDatabase fails with custom schema name

### ISSUE THREAD

**Issue by jsmarsch** [DbUp/DbUp#728]
Steps to reproduce:

- Set up a dbUp project that will call EnsureDatabase.For.SqlDatabase("someDb")
- When building the upgrader, call .JournalToSqlTable("SomeSchema", "SomeTable")
- Execute dbUp in a situation where the database does not exist and will have to be created

Observed Results
- An exception is thrown with :The specified schema name "SomeSchema" either does not exist or you do not have permission to use it."

Expected Results:
- Expected dbUp to create the schema if it does not exist in the same manner that it creates the journal table if it does not exist.

**Comment by mjauernig** [comment]
Which database?

**Comment by jsmarsch** [comment]
Yes, I'm getting the same -- with SQL Server.  Same setup as @carlsixsmith

**Comment by jsmarsch** [comment]
I think the problem is in SqlTableJournal.cs, in the CreateSchemaTableSql function.  If the custom schema does not already exist, it needs to execute a CREATE SCHEMA statement before the Create Table statement.  Do you accept pull requests?

**Comment by droyad** [comment]
See also https://github.com/DbUp/dbup-postgresql/issues/8 for Postgresql instance of this problem
